certora/conf/RecoveryModule.conf

```diff
@@ -0,0 +1,9 @@
+{
+    "files": [
+        "contracts/modules/social_recovery/SocialRecoveryModule.sol",
+    ],
+    "msg": "RecoveryModule: General Ruleset",
```

Comment:
Can we please be consistent with the naming? The contract is called `SocialRecoveryModule` however is referenced as `RecoveryModule` here, and in the file names. Basically, we should rename all files `*RecoveryModule*` -> `*SocialRecoveryModule*` and keep the naming consistent with the contract.

Reply:
Consistent naming adopted 👍🏾